acme/setup.py

```diff
@@ -20,7 +20,6 @@
     'requests',
     'setuptools',  # pkg_resources
     'six',
-    'werkzeug',
```

Comment:
With this change we need to update `letsencrypt-auto-sources/pieces/letsencrypt-auto-requirements.txt` and remove this package. Otherwise, it will continue to be installed when people use `letsencrypt-auto`.

If you'd like this to be a separate PR, that's fine but you should open an issue/PR to track the issue to make sure it makes it into 0.4.1.